services/email/index.js

```diff
@@ -50,7 +50,7 @@ Emailer.prototype.send = function send(emailConfig, callback) {
   this.transporter.sendMail({
     from: config.email.from,
     to: emailConfig.to,
-    subject: 'BRP Card',
+    subject: emailConfig.subject,
```

Comment:
Perhaps need to reconsder naming. config.email is static or from envs. emailConfig was passed. I'll update it.